### PR TITLE
Buildbot: Specify the version number of `rr_jll` that we want to use

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

**Comment by DilumAluthge** [comment]
cc: @Keno

File: commands/rr_capture.jl [hunk 1]
```diff
@@ -25,7 +25,7 @@ else
     new_env = copy(ENV)
     mktempdir() do dir
         Pkg.activate(dir)
-        Pkg.add("rr_jll")
+        Pkg.add(Pkg.PackageSpec(name = "rr_jll", version = v"5.4.1"))
```
Comment (Keno) [inline commands/rr_capture.jl:28]: I published 5.5.0+1 earlier today which fixes the issue, so let's use that.

**Comment by Keno** [comment]
Should we put the rr version in a config in the source tree instead? That way we can try out rr version bumps in a PR before enabling it on master.

**Comment by DilumAluthge** [comment]
For Buildkite, we'll be able to run CI jobs on PRs made to https://github.com/JuliaCI/julia-buildkite.

File: commands/rr_capture.jl (hunk 1 above)
Comment (DilumAluthge) [reply commands/rr_capture.jl]: Done.

**Review by Keno — APPROVED** [review]
LGTM, but it would be good to have build numbers at some point, because 5.5.0+0 does not work and we might run into similar situations again in the future. As is, this somewhat defeats the point if we push out a broken 5.5.0+2 or something.

**Comment by DilumAluthge** [comment]
> LGTM, but it would be good to have build numbers at some point, because 5.5.0+0 does not work and we might run into similar situations again in the future. As is, this somewhat defeats the point if we push out a broken 5.5.0+2 or something.

https://github.com/JuliaLang/Pkg.jl/issues/1568

**Comment by DilumAluthge** [comment]
The workaround for now, as described in https://github.com/JuliaLang/Pkg.jl/issues/1568#issuecomment-583907586 and https://github.com/JuliaLang/Pkg.jl/issues/1568#issuecomment-583907908, is to pass the full commit hash in the `rev` kwarg to the `PackageSpec`.

**Comment by DilumAluthge** [comment]
If you want, we can switch this PR to use `rev = "...specific commit hash..."`, in order to make it future-proof against e.g. a broken 5.5.0+2 in the future.

**Comment by Keno** [comment]
Eh, it's fine for the moment. Worst case we'll just suffer the broken CI for a bit.

**Pull request merged**